nodes/core/io/05-tls.html

```diff
@@ -15,17 +15,42 @@
 -->
 
 <script type="text/x-red" data-template-name="tls-config">
+    <div class="form-row" class="hide" id="node-config-row-uselocalfiles">
+        <input type="checkbox" id="node-config-input-uselocalfiles" style="display: inline-block; width: auto; vertical-align: top;">
```

Comment:
I'm not 100% sure about having this checkbox at the top like this. I wonder if there's a more inline way of doing it (whilst retaining the ability to disable local files). Not requesting any specific change... just flagging my thought on this - reserving the right to tweak after we get this merged and we get a feel for it.